deselected_tests.yaml

```diff
@@ -633,3 +633,87 @@ gpu:
   - model_selection/tests/test_validation.py::test_validation_curve_cv_splits_consistency
   - model_selection/tests/test_search.py::test_grid_search_cv_results
   - model_selection/tests/test_search.py::test_random_search_cv_results
+
+  # Segmentation faults on GPU
+  - ensemble/tests/test_forest.py::test_forest_classifier_oob
+  - ensemble/tests/test_forest.py::test_forest_regressor_oob
+  - tests/test_common.py::test_search_cv
+
+  # KMeans based (unsupported for GPU)
+  - cluster/tests/test_k_means.py
+  - tests/test_common.py::test_estimators[KMeans()
```

Comment:
```suggestion
  - tests/test_common.py::test_estimators[KMeans()]
```

Reply:
The way it is currently will deselect all configurations of pytest args that use kmeans (i.e. tests/test_common.py::test_estimators[KMeans()-check_sample_weights_invariance(kind=ones)], and tests/test_common.py::test_estimators[KMeans()-check_sample_weights_pandas_series], etc.) - around 30 in total. By closing the bracket I don't think it will detect any tests.